pkg/simulation/simulation_test.go

```diff
@@ -20,15 +21,30 @@ func TestSimulation(t *testing.T) {
 		env,
 		node.WithName("NimTest"),
 		node.WithNodePort(8000),
-		node.WithCount(10),
+		node.WithCount(5),
 	)
 	require.NoError(t, err)
 	require.NotNil(t, nodes)
 
+	done := make(chan bool)
+
 	for _, nd := range nodes {
-		l, err := nd.Logs("level")
-		assert.NoError(t, err)
-		assert.NotEmpty(t, l)
+		loch, errCh := nd.Logs()
+
+		go func() {
+			for {
+				select {
+				case ll := <-loch:
+					assert.NotEmpty(t, ll)
```

Comment:
Not enough to test this. Keep a count or the actual lines returned and check them after `<- done`